README.md

```diff
@@ -53,4 +54,14 @@ type = 0
 url = https://twitch.tv/username
 # can only be online, idle or dnd to make the rich presence work
 status = online
-```
\ No newline at end of file
+```
+
+# Termux instructions
+1. Download Termux from F-Droid (you can also download from the play store but that's outdated) 
+Optional: download hacker keyboard from playstore so that moving is easier, though, vim has touch support so only use hacker keyboard if you are going to use nano instead
```

Comment:
you could also run `mkdir $HOME/.termux/ ;echo "extra-keys = [['ESC','/','-','HOME','UP','END','PGUP'],['TAB','CTRL','ALT','LEFT','DOWN','RIGHT','PGDN']]" >> $HOME/.termux/termux.properties && termux-reload-settings && sleep 1 && logout` if you want to add arrow keys

Reply:
As I mentioned, I never needed arrow keys due to vim having native touch support.
Though, this is worth mentioning

Reply:
oh alright